lib/commands/general.js

```diff
@@ -216,6 +221,100 @@ commands.closeApp = async function () {
   await this.adb.forceStop(this.opts.appPackage);
 };
 
+commands.getDisplayDensity = async function () {
+  let out = await this.adb.shell(['getprop', 'ro.sf.lcd_density']);
+  if (out) {
+    return parseInt(out);
```

Comment:
What would you like to happen if the parse result is `NaN`? Should an error be thrown?